draft-irtf-cfrg-vdaf.md

```diff
@@ -2416,10 +2416,10 @@ of proofs.
 ## Construction {#prio3-construction}
 
 This section specifies `Prio3`, an implementation of the `Vdaf` interface
-({{vdaf}}). It has two generic parameters: an `Flp` ({{flp}}) and a `Xof`
-({{xof}}). It also has an associated constant, `PROOFS`,  with a value within
-the range of `[1, 256)`, denoting the number of FLPs generated by the Client
-({{multiproofs}}). The value of `PROOFS` is `1` unless explicitly specified.
+({{vdaf}}). It has three generic parameters: a `Field ({{field}}), an `Flp`
```

Comment:
I think we should mention the requirement that the field be FFT-friendly, and link to the relevant section, since the link was removed from the validity circuit's table of parameters.